src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,61 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.lang.Character.isDigit;
+
+public class StringAddCalculator {
+    // 메서드는 가장 작은 단위로 분리!
+    public static final String DEFAULT_DELIMITER = ",|:";
+    public static final String CUSTOM_DELIMITER_REGEXP = "//(.)\n(.*)";
+
+    public static int splitAndSum(String text) {
+        if (isBlank(text)) {
+            return 0;
+        }
+
+        Matcher m = Pattern.compile(CUSTOM_DELIMITER_REGEXP).matcher(text);
+        String customDelimiter = "";
+        if (m.find()) {
+            customDelimiter = m.group(1);
+            String[] values = split(m.group(2), customDelimiter);
+            return sum(toInts(values));
+        }
+        String[] values = split(text, DEFAULT_DELIMITER);
+        return sum(toInts(values));
+    }
+
+    private static boolean isBlank(String text) {
+        return text == null || text.isBlank();
+    }
+
+    private static String[] split(String text, String defaultDelimiter) {
+        String[] values = text.split(defaultDelimiter);
```

Comment:
```suggestion
        return text.split(defaultDelimiter);
```
로컬 변수가 의미가 없어 보이네요 바로 리턴을 해주거나
String.split 자체가 의미가 명확하기 때문에 메서드 분리 없이 splitAndSum 에서 바로 사용해도 될것 같은데
리뷰이님은 어떻게 생각하시나요 ?! 🤔

Reply:
리뷰 주신대로 변경해보니 기존의 의미를 유지하면서도 전체 소스 라인 수도 줄일 수 있었습니다!